apps/settings/locales/device_type/phone/settings.device.en-US.properties

```diff
@@ -16,8 +17,8 @@ reset-warning-body = Resetting will erase all your data.<br> Including any apps
 reset-devtools-warning-body = Resetting will erase all your data, including any apps you’ve purchased. None of your settings or data will be saved. But your phone will have unrestricted developer tools, allowing you to modify and debug system applications.
 powerSave-explanation = Turning on power save mode turns off the phone’s data, Bluetooth and Geolocation connections to extend battery life. You can still turn these services back on manually.
 performanceAndUsageDataInfo = Help us improve {{brandShortName}} by sharing data about your phone and the way you use it.
-# Add new with Help to improve firefox OS
-helpToImproveFirefoxOS =Help to improve Firefox OS
-helpToImproveFirefoxOSMessage=Help us improve Firefox OS by sharing data about your phone and the way you use it.
+# Add new with Help to improve Browser OS
+helpToImproveBrowserOS =Help to improve {{brandShortName}}
+helpToImproveBrowserOSMessage=Help us improve {{brandShortName}} by sharing data about your phone and the way you use it.
```

Comment:
After this change do we really need this string? It seems identical to the ``performanceAndUsageDataInfo`` on the line 18